deliver/lib/deliver/detect_values.rb

```diff
@@ -65,6 +65,11 @@ def find_version(options)
       elsif options[:pkg]
         options[:app_version] ||= FastlaneCore::PkgFileAnalyser.fetch_app_version(options[:pkg])
       end
+
+      puts "in detect find version"
```

Comment:
Seems like this debug bit slipped through the cracks :-)

Reply:
Haha whoooops! I still have to lint it, write tests, and do a self code review so I *may* have caught that but who knows lol 🤷‍♂️